docs/equations/parameterizations.rst

```diff
@@ -0,0 +1,197 @@
+Parameterizations
+=================
+
+pyqg support parameterizations, which are functions that take a
+:code:`pyqg.Model` and return an additional term to add to its potential
+vorticity tendency every timestep (or two terms to add to each velocity
+tendency, in which case we apply them to PV after taking their curl).
+Typically, parameterizations are used to account for the contribution of
+phenomena occuring at subgrid scales. This approach can be a computationally
+efficient way to improve the physical realism of simulations without needing to
+increase their spatial resolution (which can be very expensive).
+
+Using predefined parameterizations
+----------------------------------
+
+pyqg implements a number of predefined parameterizations (see
+:ref:`parameterizations-api` for a full list). You can use these in a
+:code:`pyqg.Model` as follows:
+
+.. code-block:: python
+
+    param = pyqg.BackscatterBiharmonic(smag_constant=0.1, back_constant=0.95)
+    model = pyqg.QGModel(parameterization=param)
+
+Note that parameterizations either target the tendencies of potential vorticity
+:math:`q` or the velocities :math:`u` and :math:`v`. If you have two
+parameterizations with the same target, you can add them together, even as a
+weighted sum. If they have different targets, you can still use both, but they
+must be passed in as separate :code:`q_parameterization` and
+:code:`uv_parameterization` arguments:
+
+.. code-block:: python
+
+    param1 = pyqg.Smagorinsky() # targets uv
+    param2 = pyqg.ZannaBolton2020() # also targets uv
+    good_model = pyqg.QGModel(parameterization=param1 + 0.25*param2) # this works!
+
+    param3 = pyqg.BackscatterBiharmonic() # targets q
+    bad_model = pyqg.QGModel(parameterization=param1 + param3) # this will error!
```

Comment:
I would prefer not to have multiple ways to pass the parameterization. Can we eliminate `parameterization=` and just force the use of `uv_parameterization` or `q_parameterization`?

Reply:
We talked on slack and decided to punt on this suggestion.